fmriprep/run_workflow.py

```diff
@@ -74,6 +67,14 @@ def main():
                          help='use ANTs-based skull-stripping')
 
     opts = parser.parse_args()
+    create_workflow(opts)
+
+
+def create_workflow(opts):
+    import logging
+    from nipype import config as ncfg
+    from fmriprep.workflows import base as fwb
+    from fmriprep.workflows.base import base_workflow_enumerator
```

Comment:
I've been meaning to ask this--how come sometimes we import at the top of the file, and other times inside the function?